bin/improver-wind-downscaling

```diff
@@ -95,7 +95,6 @@ def main():
     else:
         veg_roughness_cube = None
     try:
-        realization_coord = wind_speed.coord('realization')
```

Comment:
I have tested that the slices_over('realizations') creates the same error as coord('realization') so this is ok to remove